src/overlays/actors/ovl_En_Elf/z_en_elf.c

```diff
@@ -1083,7 +1083,8 @@ void func_80A0461C(EnElf* this, PlayState* play) {
     } else {
         arrowPointedActor = play->actorCtx.targetCtx.arrowPointedActor;
 
-        if ((player->stateFlags1 & PLAYER_STATE1_10) || ((YREG(15) & 0x10) && func_800BC56C(play, 2))) {
+        if ((player->stateFlags1 & PLAYER_STATE1_10) ||
+            ((R_SCENE_CAM_TYPE & SCENE_CAM_TYPE_SHOP) && Play_CheckViewpoint(play, VIEWPOINT_PIVOT))) {
```

Comment:
```suggestion
            ((R_SCENE_CAM_TYPE & 0x10) && Play_CheckViewpoint(play, VIEWPOINT_PIVOT))) {
```
this checks `SCENE_CAM_TYPE_SHOP` or `SCENE_CAM_TYPE_FIXED` .....
(why did they write code like this??)

Reply:
I'm not following this comment, not sure if you're suggesting to actually remove the enum? Or where `SCENE_CAM_TYPE_FIXED` comes from?

Reply:
& 0x10 will be true with both `SCENE_CAM_TYPE_SHOP` and `SCENE_CAM_TYPE_FIXED`, cause its doing a bitwise `and`. Issue its its doing a bitwise `and` on something that isnt a flag, so its incredibly confusing. 

Just using `SCENE_CAM_TYPE_SHOP` as you have it currently is not correct, cause it passes for two different types. |
You could theoretically do `SCENE_CAM_TYPE_SHOP | SCENE_CAM_TYPE_FIXED`, but that still gives the wrong idea cause these arent bit flags. It only works cause the second one is zero lol

Reply:
because this is a biwise `and`, it passes for both `SCENE_CAM_TYPE_SHOP` and `SCENE_CAM_TYPE_FIXED`. They both have that bit set within their values of 0x10 and 0x30

So as you have it now, it is misleading cause it makes it seem like it is only about shops.
The issue is more so rooted in them doing a bitwise `and` on values they otherwise dont treat as bitflags. But it is curious that they chose the values that they did (0x10, 0x20, 0x30, 0x40)

Reply:
inb4 they are flags and 0x30 is the bitflag for fixed |'d with a special flag for shops lmao 
the rest are single bits so..

Reply:
Oh you're right, yeah I didn't even notice or think of that... Yeah I see why this is very strange code. It's only like this for fairy so not exactly sure what they were going for here...

Reply:
just do `R_SCENE_CAM_TYPE & SCENE_CAM_TYPE_SHOP & SCENE_CAM_TYPE_FIXED` :)